app/src/main/java/swati4star/createpdf/database/History.java

```diff
@@ -0,0 +1,57 @@
+package swati4star.createpdf.database;
+
+import android.arch.persistence.room.ColumnInfo;
+import android.arch.persistence.room.Entity;
+import android.arch.persistence.room.PrimaryKey;
+
+@Entity
+public class History {
+    @PrimaryKey(autoGenerate = true)
+    private int mId;
+    @ColumnInfo(name = "file_name")
+    private String mFileName;
+
+    @ColumnInfo(name = "date")
+    private String mDate;
+
+    @ColumnInfo(name = "operation_type")
+    private String mOperationType;
+
+    public History(String fileName, String date, String operationType) {
+        this.mFileName = fileName;
```

Comment:
instead of filename. Store file path/uri, so that we can later perform more operations to it.
For example, opening the file

Reply:
And for the deleted file?

Reply:
Don't do anything for the deleted file.